qtt/instrument_drivers/virtualAwg/awgs/ZurichInstrumentsHDAWG8.py

```diff
@@ -4,7 +4,7 @@
 
 class ZurichInstrumentsHDAWG8(AwgCommon):
     __sampling_rate_map = {0: 2.4e9, 1: 1.2e9, 2: 600e6, 3: 300e6, 4: 150e6, 5: 72e6, 6: 37.50e6, 7: 18.75e6,
-                           8: 9.4e6, 9: 4.5e6, 10: 2.34e6, 11: 1.2e3, 12: 586e3, 13: 293e3}
+                           8: 9.4e6, 9: 4.5e6, 10: 2.34e6, 11: 1.2e6, 12: 586e3, 13: 293e3}
```

Comment:
@CumulonimbusCalvus Was this indeed a typo?

Reply:
Yes, but there where some more sample_rate value typo's. I updated the `sampling_rate_map` to the correct values.